src/NUnitFramework/framework/Constraints/ExactCountConstraint.cs

```diff
@@ -31,9 +31,19 @@ namespace NUnit.Framework.Constraints
     /// item in a collection, succeeding only if a specified
     /// number of items succeed.
     /// </summary>
-    public class ExactCountConstraint : PrefixConstraint
+    public class ExactCountConstraint : Constraint
     {
-        private int expectedCount;
+        private readonly int expectedCount;
+        private readonly IConstraint itemConstraint;
```

Comment:
Our new coding standard is for member variables to start with an underscore. We don't require that submitters update every file they touch, but if you are adding new members or making large changes to a file, refactoring to `_expectedCount` and adding `_itemContraint` would be nice.